libs/ledger/include/ledger/protocols/main_chain_rpc_service.hpp

```diff
@@ -91,6 +91,8 @@ class MainChainRpcService : public muddle::rpc::Server,
   MainChainRpcService(MainChainRpcService &&)      = delete;
   ~MainChainRpcService() override                  = default;
 
+  void Start();
```

Comment:
Only attach the gossip listener after setting up the main chain + genesis, it is pointless otherwise.